<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DropFileWidget</class>
 <widget class="QWidget" name="DropFileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>244</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>244</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>244</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#container,
#fileProcessor {
	background-color: white;
	border-radius: 8px;	
	border: 1px solid #34343426;
}

QToolButton {
	background: transparent;
	border: none;
}

#label {
	font-family: Open Sans;
	font-size: 18px;
	font-weight: 500;
	line-height: 26px;
	color: #1A1A1A;
}

#fileName {
	font-family: Open Sans;
	font-size: 16px;
	font-weight: 500;
	line-height: 24px;
	color: #1A1A1A;
}

#fileSize{
	font-family: Open Sans;
	font-size: 12px;
	font-weight: 600;
	line-height: 20px;
	color:#666666;
}

QProgressBar {
    border: none;
    border-radius: 4px;
    background-color: #EEF4F6;
    font-family: Open-Sans;
    font-weight: bold;
    font-size: 14px;
    color: #333;
    height: 24px;
}

QProgressBar::chunk {
    background-color: #22084B;
    border-radius: 4px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="container" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>146</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>146</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="containerLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="topVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>35</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="fileSelectButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>87</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>87</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../../Resources/Resources.qrc">
          <normaloff>:/images/Automations/icon_drag_drop.svg</normaloff>:/images/Automations/icon_drag_drop.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>87</width>
          <height>87</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="uploadTipLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string>Drag &amp; Drop your file here</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="bottomVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>36</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="fileProcessor" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>90</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="filetypeIcon">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>65</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../../Resources/Resources.qrc">
          <normaloff>:/images/Automations/icon_upload_file.svg</normaloff>:/images/Automations/icon_upload_file.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>65</width>
          <height>65</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="fileInfoLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="fileNameLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Report_pop_123</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="closeButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Resources/Resources.qrc">
              <normaloff>:/images/Automations/icon_upload_close.svg</normaloff>:/images/Automations/icon_upload_close.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="fileSizeLabel">
          <property name="text">
           <string>12MB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>8</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>8</height>
           </size>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
